libsession/src/main/java/org/session/libsession/snode/SnodeAPI.kt

```diff
@@ -530,7 +530,7 @@ object SnodeAPI {
             400, 500, 502, 503 -> { // Usually indicates that the snode isn't up to date
                 handleBadSnode()
             }
-            406 -> {
+            425 -> {
```

Comment:
I wonder if we will need to maintain both of these for this class of errors

Reply:
Yes, going to revert this bit.